.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+**Description:**
+
+Describe in a couple of sentences what this PR adds
+
+**JIRA:**
+
+[XXX-XXXX](https://openedx.atlassian.net/browse/XXX-XXXX)
+
+**Additional Details**
+
+* **Dependencies:**: List dependencies on other outstanding PRs, issues, etc.
+* **Merge deadline:** List merge deadline (if any)
+* **Testing instructions:** Provide non-trivial testing instructions
+* **Author concerns:** List any concerns about this PR
+
+**Reviewers:**
+- [ ] @edx/arch-review
+- [ ] tag reviewer
+
+**Merge checklist:**
+- [ ] All reviewers approved
+- [ ] CI build is green
+- [ ] Version bump if needed
+- [ ] Changelog record added
```

Comment:
**Notes for Reviewers**: This is what I really care about.  Happy to kill the rest of these updates, but they are modified from the cookie cutter version.